base/reduce.jl

```diff
@@ -396,7 +396,11 @@ end
 
 
 ## prod
+"""
+    prod(f, iter)
```

Comment:
Only what's written in the `@docs` blocks matters now. Signature headers in docstrings aren't used like they were with `genstdlib.jl`. For the `prod` docstring the simplest would be to do

```diff
diff --git a/doc/src/stdlib/collections.md b/doc/src/stdlib/collections.md
index 573332a..4b1defa 100644
--- a/doc/src/stdlib/collections.md
+++ b/doc/src/stdlib/collections.md
@@ -116,8 +116,7 @@ Base.sumabs!
 Base.sumabs2(::Any)
 Base.sumabs2(::Any, ::Any)
 Base.sumabs2!
-Base.prod(::Any)
-Base.prod(::Any, ::Any)
+Base.prod
 Base.prod!
 Base.any(::Any)
 Base.any(::AbstractArray, ::Any)
```

and then all three docstrings (and any future ones) will just get concatenated there.

Other option would be to add a new signature for it:

```diff
diff --git a/doc/src/stdlib/collections.md b/doc/src/stdlib/collections.md
index 573332a..41a74c0 100644
--- a/doc/src/stdlib/collections.md
+++ b/doc/src/stdlib/collections.md
@@ -118,6 +118,7 @@ Base.sumabs2(::Any, ::Any)
 Base.sumabs2!
 Base.prod(::Any)
 Base.prod(::Any, ::Any)
+Base.prod(::Base.Callable, ::Any)
 Base.prod!
 Base.any(::Any)
 Base.any(::AbstractArray, ::Any)
```